src/components/templates/descopeMiddleware.tsx

```diff
@@ -57,16 +57,30 @@ export const DescopeMiddleware = ({ children }: { children: ReactNode }) => {
 				const error = await getLoggedInUser();
 				if (error) {
 					addToast({
-						message: t("errors.loginFailed", { error }),
+						message: t("errors.loginFailedExtended", { error }),
```

Comment:
Why message Extended inside toast?

Reply:
done

Reply:
also removed {error} from the translation function.

Reply:
Thank you